app/controllers/PersonalDetailsController.scala

```diff
@@ -116,21 +118,21 @@ class PersonalDetailsController(applicationClient: ApplicationClient,
     case RedirectToTheDashboard => false
   }
 
-  private def submit(generalDetailsForm: Form[GeneralDetailsForm.Data], onSuccess: OnSuccess, redirectOnSuccess: Result,
+  private def submit(generalDetailsForm: Form[PersonalDetailsForm.Data], onSuccess: OnSuccess, redirectOnSuccess: Result,
```

Comment:
Do we need to retrieve edipCompleted param from cache for sdip  and submit here otherwise we may override this to None if the candidate edits the personal details after submitting application.